docs/modules/ROOT/pages/how-tos/cloudscale/update_compute_flavors.adoc

```diff
@@ -0,0 +1,185 @@
+
+git checkout -b update-compute-flavors
+
+yq eval -i ".parameters.openshift4_terraform.terraform_variables.infra_flavor = \"${INFRA_FLAVOR}\"" \
+  ${CLUSTER_ID}.yml
+yq eval -i ".parameters.openshift4_terraform.terraform_variables.worker_flavor = \"${WORKER_FLAVOR}\"" \
+  ${CLUSTER_ID}.yml
+yq eval -i ".parameters.openshift4_terraform.terraform_variables.additional_worker_groups.storage.flavor = \"${STORAGE_FLAVOR}\"" \
+  ${CLUSTER_ID}.yml
+----
+
+Ensure at least version `3.12` of https://github.com/appuio/terraform-openshift4-cloudscale[terraform-openshift4-cloudscale] is used to make sure the flavors for master and loadbalancer nodes get updated as well.
+
+. Commit and create MR to review
++
+[source,bash]
+----
+git commit -a -m "Update compute flavors on cluster ${CLUSTER_ID}"
+git push -u origin update-compute-flavors
+----
+
+. Compile and push the cluster catalog.
+
+== Prepare Terraform environment
+
+include::partial$commodore-init.adoc[]
+
+. Configure Terraform environment
++
+[source,bash]
+----
+export CLOUDSCALE_API_TOKEN=
+export GITLAB_USER=
+export GITLAB_TOKEN=
+----
+
+include::partial$cloudscale/configure-terraform-secrets.adoc[]
+
+. Setup Terraform
++
+.Prepare Terraform execution environment
+[source,bash]
+----
+# Set terraform image and tag to be used
+tf_image=$(\
+  yq eval ".parameters.openshift4_terraform.images.terraform.image" \
+  dependencies/openshift4-terraform/class/defaults.yml)
+tf_tag=$(\
+  yq eval ".parameters.openshift4_terraform.images.terraform.tag" \
+  dependencies/openshift4-terraform/class/defaults.yml)
+
+# Generate the terraform alias
+base_dir=$(pwd)
+alias terraform='docker run --rm \
+  -e REAL_UID=$(id -u) \
+  --env-file ${base_dir}/terraform.env \
+  -w /tf \
+  -v $(pwd):/tf \
+  --ulimit memlock=-1 \
+  "${tf_image}:${tf_tag}" /tf/terraform.sh'
+
+export GITLAB_REPOSITORY_URL=$(curl -sH "Authorization: Bearer $(commodore fetch-token)" ${COMMODORE_API_URL}/clusters/${CLUSTER_ID} | jq -r '.gitRepo.url' | sed 's|ssh://||; s|/|:|')
+export GITLAB_REPOSITORY_NAME=${GITLAB_REPOSITORY_URL##*/}
+export GITLAB_CATALOG_PROJECT_ID=$(curl -sH "Authorization: Bearer ${GITLAB_TOKEN}" "https://git.vshn.net/api/v4/projects?simple=true&search=${GITLAB_REPOSITORY_NAME/.git}" | jq -r ".[] | select(.ssh_url_to_repo == \"${GITLAB_REPOSITORY_URL}\") | .id")
+export GITLAB_STATE_URL="https://git.vshn.net/api/v4/projects/${GITLAB_CATALOG_PROJECT_ID}/terraform/state/cluster"
+
+pushd catalog/manifests/openshift4-terraform/
+----
++
+.Initialize Terraform
+[source,bash]
+----
+terraform init \
+  "-backend-config=address=${GITLAB_STATE_URL}" \
+  "-backend-config=lock_address=${GITLAB_STATE_URL}/lock" \
+  "-backend-config=unlock_address=${GITLAB_STATE_URL}/lock" \
+  "-backend-config=username=${GITLAB_USER}" \
+  "-backend-config=password=${GITLAB_TOKEN}" \
+  "-backend-config=lock_method=POST" \
+  "-backend-config=unlock_method=DELETE" \
+  "-backend-config=retry_wait_min=5"
+----
```

Comment:
```suggestion
+
include:partial$setup_terraform.adoc[]
```

Reply:
The `terraform` alias is slightly different. This is necessary for all the commands to work so we can't use the include.

Reply:
There would also be the option to use page variables to influence such differences. But yea, it might be questionable if it is worth the effort.